TestingLowerBounds/FDiv/CondFDiv.lean

```diff
@@ -471,6 +472,13 @@ lemma condFDiv_zero_measure : condFDiv f κ η 0 = 0 := by
   rw [condFDiv_eq' hf_ae integrable_zero_measure]
   simp only [integral_zero_measure, EReal.coe_zero]
 
+@[simp]
+lemma condFDiv_isEmpty_left [IsEmpty α] : condFDiv f κ η μ = 0 := by
```

Comment:
If the `right` version of that lemma makes no sense, consider removing the `left` form the name. Also you should add an `_of_`:
```suggestion
lemma condFDiv_of_isEmpty_left [IsEmpty α] : condFDiv f κ η μ = 0 := by
```

Reply:
I will add `right` version in a minute